src/components/navbar/navLinks/navLinks.js

```diff
@@ -0,0 +1,34 @@
+import history from '../../../utils/history';
+import routePaths from '../../../constants/routePaths';
+
+
+const NavLinks = () => {
+
+    const links = [
+        {
+          path: 'home',
```

Comment:
please have your paths be in the pattern of shopping-cart instead of camel case

Reply:
The sting in the "path" property pulls from the routePaths const to get the actual URL path in the correct "path-name-here" format. Is that too redundant?

Reply:
That sounds correct

Reply:
Ex: path: routePaths.home